tests/test/test_checkout_volunteer.py

```
@@ -1,20 +1,39 @@
 import pytest
-from tests.pages.checkout_page import CheckOutPage
-from tests.utilities.fixtures import login_with_volunteer
+
+from tests.pages.home_page import HomePage
 from tests.utilities.fixtures import driver
+from tests.utilities.fixtures import login_with_volunteer
+from tests.pages.checkout_page import CheckOutPage
 
+@pytest.fixture(scope="function")
+def home_page(driver):
```

Ruff (0.14.14)

Redefinition of unused `driver` from line 4: `driver` redefined here

(F811)